authMiddleware.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = (req, res, next) => {
+  const token = req.header('auth-token');
+  if (!token) return res.status(401).send('Access Denied');
+
```

Comment:
Returning the response immediately after finding no token might not be the best approach. Consider throwing an error and catching it in a centralized error handler for more consistent error handling.